dist/api_data.py

```diff
@@ -659,6 +659,12 @@ def __ge__(self, other):
                 If no in-memory ref is found on the root page, attempt to locate a random
                 in-memory page by examining all entries on the root page.''',
                 type='boolean'),
+            Config('evict_use_npos', 'false', r'''
```

Comment:
What does npos mean?

Reply:
Added clarification in description.

Reply:
Why just we call it normalized positions all the time?